coralproject/cay
# Copy Component

# Copy Component

Well... basically. CopyToClipboard Component wasn't working **AT ALL** no matter what I try. So, I've built our component with one of the most popular github repos (clipboard.js) Which happens to be smaller.

The usage is pretty simple. I'll extend the API in the future, but for now is more than enough.

``` javascript
<Copy
  target="textarea#standaloneCode"
  onCopy={()=> this.showCopied('standalone')}
  >
  <CoralButton>
    Copy
  </CoralButton>
</Copy>
```
## What does this PR do?
- Replaces CopyToClipboard Component library for Copy Component in our own components.
## How do I test this PR?
- Go to a form
- Click Publish Options and use the copy buttons
- Then must work as expected.

@coralproject/frontend
